src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), each property should be on its own line. The 'animation' shorthand should be expanded into its individual properties for better readability.